### ISSUE TITLE
Use Django's built-in bulk_update method when available

### ISSUE THREAD

**Issue by budlight** [Tesorio/django-anon#48]
https://docs.djangoproject.com/en/3.0/ref/models/querysets/#bulk-update

I could not get this to work with bulk_update, however

```
            self._meta.model.objects.bulk_update(
                objs,
                update_fields
            )
```

worked fine

**Comment by caioariede** [comment]
@budlight could you please provide the traceback for the error you were getting?

**Comment by budlight** [comment]
```
Traceback (most recent call last):
  File "./manage.py", line 21, in <module>
    main()
  File "./manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/src/hsb/management/commands/anonymize_database.py", line 14, in handle
    hsb.anon.UserAnonymizer().run(select_chunk_size=20, batch_size=20)
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/src/django-anon/anon/base.py", line 92, in run
    **bulk_update_kwargs
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django_bulk_update/helper.py", line 174, in bulk_update
    pk_value, _ = _as_sql(obj, pk_field, query, compiler, connection)
  File "/root/.local/share/virtualenvs/src-NVTF7jWz/lib/python3.7/site-packages/django_bulk_update/helper.py", line 24, in _as_sql
    value = getattr(obj, field.attname)
AttributeError: 'User' object has no attribute 'user_ptr_id'
```

**Comment by budlight** [comment]
It generates some SQL that is abhorent, I would say it really should probably use the builtin bulk_update by default and optionally use the other.

**Comment by caioariede** [comment]
@budlight would you be able to provide a small example that can reproduce the issue?

In the traceback you provided it seems that it can be related to model inheritance

It’s likely a good idea to use Django’s native bulk update when it’s available but we still want to address the issue with the current implementation for older Django versions that does not provide the built in bulk update support

**Comment by caioariede** [comment]
I'd love to move forward with this!

For anyone willing to contribute, here's my proposal / what I think would be a good solution:

1. Create a function `bulk_update` inside a `compat.py` file
2. This function should use Django's original `bulk_update` implementation if available, otherwise the [django-bulk-update package](https://github.com/aykut/django-bulk-update)
3. Update `base.py` to import `bulk_update` from the `compat` module

How to contribute
------------------

We have a good documentation here: https://django-anon.readthedocs.io/en/latest/contributing.html

**Comment by caioariede** [comment]
Fixed in #56
